<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyPalletizeForm</class>
 <widget class="GUIFormBase" name="PropertyPalletizeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonEditLibrary">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ライブラリ編集</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>表示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonShowPallet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>パレット領域</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonShowWork">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>検出ワーク／側面</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetControl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>XPos</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>YPos</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Dx</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Dy</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Raw</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Converted</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>送信</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>XPos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>YPos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rotation</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dx</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dy</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Index</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBoxRegIndex">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>37</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
